### ISSUE TITLE
Game doesn't allow me to place a tile next to an open lake edge

### ISSUE THREAD

**Issue by nightpool** [ondras/rri#6]
I think I should be able to play the highlighted tile in the square I marked?

![image](https://user-images.githubusercontent.com/233815/83956527-0fc84800-a82d-11ea-9cde-a37e0f625f7a.png)

**Comment by nightpool** [comment]
(I mean, i don't think it's a _good_ idea, but it seems like it should be possible?

**Comment by ondras** [comment]
Hmm, good question. My current opinion is that such placement is *not* allowed, but perhaps the Lake rulebook states otherwise?

**Comment by bcsak** [comment]
I think that kind of drawing is not possible.
You can find the following in the [Rulebook/Drawing Routes](https://www.horrible-games.com/wp-content/uploads/2018/07/RRI_BLU_Rulebook_ENG_v05_light.pdf) section:
- You can’t draw Routes in a way that directly connects Railways to Highways or vice versa (you need a Station to do that).

You have to take account of the Lake tiles as the other connections as well. So, a "Station" is necessary.

**Comment by nightpool** [comment]
no, there is no reason to assume that by that rule, the designers also meant to forbid placing an open lake face adjacent to a railroad. That rule specifically says that it only applies to Railroads and Highways, there's nothing in the rules that applies that to a lake.

**Comment by apcshields** [comment]
See this thread: https://boardgamegeek.com/thread/2439612/question-about-lakes

The publisher appears to confirm @nightpool‘s interpretation.

**Comment by ondras** [comment]
> The publisher appears to confirm @nightpool‘s interpretation.

...making this issue a valid bugreport!
